src/lib/merkle_mask/maskable_merkle_tree.ml

```diff
@@ -143,15 +143,20 @@ module Make (Inputs : Inputs_intf) = struct
     attached_mask
 
   let unregister_mask_exn (t : t) (mask : Mask.Attached.t) =
-    let error_msg = "unregister_mask: no such registered mask" in
+    let error_msg_prefix =
+      sprintf "Couldn't unregister mask with UUID %s from parent %s"
+        (Mask.Attached.get_uuid mask |> Uuid.to_string_hum)
+        (get_uuid t |> Uuid.to_string_hum)
+    in
     let t_uuid = get_uuid t in
```

Comment:
teensy nit: could use `t_uuid` above instead of `get_uuid t` in the calculation of `error_msg_prefix`